platformcomponents/desktop/content-card-item.json

```diff
@@ -0,0 +1,20 @@
+{
+  "content-card-item": {
+    "comment": "Applied to file attachments from Content tab",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49650%3A108097",
+    "border": "@theme-outline-secondary-normal",
+    "iconForeground": "@theme-common-text-white",
+    "iconBackground": {
+      "default": "@theme-background-accent-normal"
```

Comment:
you do not need to add the other colours for this preview box? like red, orange, green, etc? if you don't need it to complete your work, it is ok not to add it